.appveyor.yml

```diff
@@ -54,6 +54,8 @@ install:
     set PULUMI_LOCAL_NUGET=C:\Pulumi\nuget
     
     set PULUMI_TEST_OWNER=moolumi
+
+    choco install dep
```

Comment:
This won’t necessarily get the same version we use for testing on other platforms. Should we force getting a specific version?

Reply:
This is now redundant as this is going away since we moved to GHA